### ISSUE TITLE
Missing period SA1629 should not trigger inside <example> tags.

### ISSUE THREAD

**Issue by PatrickBig** [DotNetAnalyzers/StyleCopAnalyzers#3804]
The `<example>` tag is often used to show a literal value which may be shown in OpenAPI specs or other documentation.

```csharp
/// <summary>
/// Message containing details needed to create a new customer record.
/// </summary>
public class CreateCustomerRequest
{
    /// <summary>
    /// Gets or sets the name of the customer as shown in their profile.
    /// </summary>
    /// <example>
    /// JOHN SMITH
    /// </example>
    public string CustomerName { get; set; }

   /// more properties......
}
```

The comment "JOHN SMITH" above the `CustomerName` property would trigger a SA1629 code unless we add an exclusion for it in some way.

I believe it would be helpful if we could disable it for the example tag, or perhaps make it configurable.

**Comment by PatrickBig** [comment]
You're right that it should represent a full documentation section, but I never tried wrapping it in `code`.

I was mostly concerned with my usage in Swagger, but I got that sorted out. As long as the `<example>` tag ONLY contains a `<code>` tag it will still render it properly.

I'm fine with closing. Thank you for the tip.
